tests/unit/mb_enhanced_cover_art_uploads/providers/datpiff.test.ts

```diff
@@ -52,7 +52,7 @@ describe('datpiff provider', () => {
                 types: [ArtworkTypeIDs.Back],
             }],
         }, {
-            desc: 'mixtape without front cover',
+            desc: 'mixtape without back cover',
```

Comment:
One of the ugliest places to change a string, the WARC file also has to be renamed now...

Reply:
Oops, totally forgot about that. It looked like such a trivial change